store/src/java/com/zimbra/cs/service/UserServlet.java

```diff
@@ -405,6 +404,8 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
         } catch (ServiceException se) {
             if (se.getCode() == ServiceException.PERM_DENIED || se instanceof NoSuchItemException)
                 sendError(context, req, resp, L10nUtil.getMessage(MsgKey.errNoSuchItem, req));
+            else if (se.getCode() == ServiceException.AUTH_REQUIRED)
```

Comment:
Personally, I like to wrap if statements in `{}` it makes code more readable.

Reply:
Agreed. I just didn't at first to follow the style of the existing block around it, but just changed it to `{}` wrapped statements as per your suggestion.

Done.